README.md

```diff
@@ -6,33 +6,25 @@
 
 Ruby client for the [Unleash](https://github.com/Unleash/unleash) feature management service.
 
-- [Supported Ruby Interpreters](#supported-ruby-interpreters)
+>  **Migrating to v6** 
```

Comment:
Since it's kind of a temporary announcement, I think it makes sense to use a callout, instead of a regular heading, and put it on top for visibility.

Reply:
Awesome! I'm happy if you are